subsys/config/common/src/app_subGHz_config.c

```diff
@@ -138,6 +131,26 @@ const radio_sx126x_device_config_t radio_sx1262_cfg = {
 
 const radio_sx126x_device_config_t *get_radio_cfg(void)
 {
+	radio_sx1262_cfg.gpio_power =
+		sid_gpio_utils_get_gpio_number_dt(
+			(struct gpio_dt_spec)GPIO_DT_SPEC_GET(DT_NODELABEL(semtech_sx1262_reset_gpios), gpios));
```

Comment:
Use [GPIO_DT_SPEC_GET_OR](https://docs.zephyrproject.org/apidoc/latest/group__gpio__interface.html#ga26b2205aa82819df1d80a22761352e71) then undefined GPIO will be set as e.g. `GPIO_UNUSED_PIN`, and finally you can change line 152 to more generic.
```suggestion
			(struct gpio_dt_spec)GPIO_DT_SPEC_GET_OR(DT_NODELABEL(semtech_sx1262_reset_gpios), gpios), GPIO_UNUSED_PIN);
```

Reply:
OK